eyeshade/workers/reports.js

```diff
@@ -17,6 +17,7 @@ let altcurrency
 
 const datefmt = 'yyyymmdd-HHMMss'
 const datefmt2 = 'yyyymmdd-HHMMss-l'
+const feePercent = 0.05
```

Comment:
it's good to remove the constant from the running code... but isn't it better to get it from the config and have `initialize` set it with something like

        feePercent = runtime.config.processingFee || 0.05

Reply:
somewhat, but this is a `const` whereas I think following that pattern allows for accidental reassignment and I don't think we'll change the processing fee soon